src/main/kotlin/lotto/ticket/LottoNumber.kt

```diff
@@ -5,8 +5,15 @@ data class LottoNumber(private val number: Int) {
     companion object {
         private const val MIN_NUMBER = 1
         private const val MAX_NUMBER = 45
-        val LOTTO_NUMBER_BOX: Map<Int, LottoNumber> =
-            (MIN_NUMBER..MAX_NUMBER).map { it to LottoNumber(it) }.toMap()
+
+        val LOTTO_NUMBER_BOX: Map<Int, LottoNumber> = (MIN_NUMBER..MAX_NUMBER)
+            .map {
+                it to LottoNumber(it)
+            }.toMap()
```

Comment:
associateBy도 활용해보실 수 있겠네요.

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.sequences/associate-by.html

```suggestion
val LOTTO_NUMBER_BOX: Map<Int, LottoNumber> = (MIN_NUMBER..MAX_NUMBER).associateBy({ it }, { LottoNumber(it) })
```